hydra-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -282,6 +283,13 @@ spec = around (showLogsOnFailure "EndToEndSpec") $ do
               publishHydraScriptsAs node Faucet
                 >>= threeNodesNoErrorsOnOpen tracer tmpDir node
 
+      it "node can support multiple etcd clusters" $ \tracer ->
+        failAfter 60 $
+          withClusterTempDir $ \tmpDir -> do
+            withCardanoNodeDevnet (contramap FromCardanoNode tracer) tmpDir $ \node -> do
+              publishHydraScriptsAs node Faucet
```

Comment:
Should: not need a cardano node and published scripts for this test (use offline chain)